### PR TITLE
lambda-delta: init at 0.98.3

### PR DESCRIPTION
**Pull request by siraben**
I don't know how to use the software, but it builds.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/misc/emulators/lambda-delta/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, autoreconfHook, pkg-config, SDL2 }:
```
Comment (IvarWithoutBones) [inline pkgs/misc/emulators/lambda-delta/default.nix:1]: `lib` isn't used.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -22028,6 +22028,8 @@ julia_15 = callPackage ../development/compilers/julia/1.5.nix {
 
   kvirc = libsForQt514.callPackage ../applications/networking/irc/kvirc { };
 
+  lambda-delta = callPackage ../development/misc/emulators/lambda-delta { };
```
Comment (IvarWithoutBones) [inline pkgs/top-level/all-packages.nix:22031]: Path is incorrect.

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 104276` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>lambda-delta</li>
  </ul>
</details>

**Comment by siraben** [comment]
@SuperSandro2000 Looks good to merge?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104276` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>lambda-delta</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104276` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>lambda-delta</li>
  </ul>
</details>

**Pull request merged**